activejob/lib/active_job/railtie.rb

```diff
@@ -25,6 +25,16 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_job.enqueue_after_transaction_commit" do |app|
```

Comment:
Why aren't we letting the next initialize do it?

Reply:
The next initializer sets things on `ActiveJob`, not `ActiveJob::Base`. We need the later because it's best if this feature is configurable as an inherited class variable. It allows engines and apps to have a base job class with the configuration they want and that differ from the rest of the app.

Reply:
It does in both. See lines 66-67

Reply:
Ah my bad. Either way the property only exists if the module is included, so only if Active Record is loaded. So now there is no way we can use the general initializer. No big deal.